src/components/dataset/query/sidebar/filter/CategoryFilter.jsx

```diff
@@ -24,14 +24,14 @@ export class CategoryFilter extends React.PureComponent {
 
   static propTypes = {
     column: PropTypes.object,
-    count: PropTypes.string,
+    count: PropTypes.number,
     filterData: PropTypes.object,
     handleChange: PropTypes.func,
     name: PropTypes.string,
     table: PropTypes.string,
   };
 
-  componentWillReceiveProps(nextProps) {
+  UNSAFE_componentWillReceiveProps(nextProps) {
```

Comment:
Is the UNSAFE tag something that was added automatically? What caused this change?

Reply:
It's recommended we switch to [getDerivedStateFromProps](https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops) instead of this. Let's try it out with this PR.